fastai/callback/mixup.py

```diff
@@ -7,15 +7,20 @@
 from torch.distributions.beta import Beta
 
 # Cell
-def reduce_loss(loss, reduction='mean'):
+def reduce_loss(
+    loss:Tensor,
+    reduction:(str, None)='mean' # Loss reduction type
```

Comment:
Would it be good to list all options here since there are only three?

Reply:
Or could state:
```suggestion
    reduction:str='mean' # PyTorch loss reduction
```
Matching PyTorch, the type should only be str

Reply:
Agree with @warner-benjamin on this one.